UPGRADE-1.14.md

```diff
@@ -1036,6 +1032,10 @@
     * `sylius.{user_type}_user.pin_generator.password_reset`
     * `sylius.{user_type}_user.pin_uniqueness_checker.password_reset`
 
+## Miscellaneous
+
+1. Starting with Sylius 1.14, form types will be extended using the parent form instead of through form extensions, like it's done in the `Sylius\Bundle\AdminBundle\Form\Type\CatalogPromotionScopeType` and `Sylius\Bundle\AdminBundle\Form\Type\CatalogPromotionActionType` classes.
+
```

Comment:
```suggestion
## Miscellaneous

1. Starting with Sylius 1.14, form types will be extended using the parent form instead of through form extensions, like it's done in the `Sylius\Bundle\AdminBundle\Form\Type\CatalogPromotionScopeType` and `Sylius\Bundle\AdminBundle\Form\Type\CatalogPromotionActionType` classes.

```
IMO there's too little info in here.
Also don't we have, or rather should, an architecture changes section?